src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -179,6 +179,18 @@ private IOperation CreateInternal(BoundNode boundNode)
                     return CreateBoundInterpolationOperation((BoundStringInsert)boundNode);
                 case BoundKind.LocalFunctionStatement:
                     return CreateBoundLocalFunctionStatementOperation((BoundLocalFunctionStatement)boundNode);
+                case BoundKind.ConstantPattern:
+                    return CreateBoundConstantPatternOperation((BoundConstantPattern)boundNode);
+                case BoundKind.DeclarationPattern:
+                    return CreateBoundDeclarationPatternOperation((BoundDeclarationPattern)boundNode);
+                case BoundKind.WildcardPattern:
+                    return null;
```

Comment:
I will do this change in https://github.com/dotnet/roslyn/pull/20364, I forgot to add this commit to this PR.